torch/_subclasses/fake_tensor.py

```diff
@@ -56,6 +57,17 @@ def _is_tensor_constructor(func: OpOverload):
         len(schema.returns) == 1 and schema.returns[0].type is torch._C.TensorType.get()
     )
 
+cpu_fallback_enabled = False
+
+@contextlib.contextmanager
+def enable_cpu_fallback(enable_cpu_fallback: bool):
+    global cpu_fallback_enabled
+    orig = cpu_fallback_enabled
+    cpu_fallback_enabled = enable_cpu_fallback
+    try:
+        yield
+    finally:
+        cpu_fallback_enabled = orig
```

Comment:
Why not have this on the fake tensor mode itself?

Reply:
I guess... is this something that we view as temporary? If so, it might be nice not to commit it to the FakeTensorMode api. But I guess it's not that different either way.. I can move into the Mode

Reply:
It is simpler and organizationally better for it to live on the mode, so yes I'd prefer it!

Re how tensors can get to it, that would be done by storing the mode on the tensor!

Reply:
Converter should have a weak ref to the input tensor